docs/pages/docs/connectors/injected.en-US.mdx

```diff
@@ -25,6 +25,20 @@ const connector = new InjectedConnector()
 
 ## Configuration
 
+### allowConnectToUnsupportedChain
```

Comment:
Maybe we should call this `switchChainOnConnect` instead to more accurately represent what this does?

`allowConnectToUnsupportedChain` implies that someone can't switch to unsupported chains, but most wallet apps will still allow this through their interface.